packages/testing/scripts/download-test-server.mjs

```diff
@@ -20,64 +40,28 @@ try {
 
 const pipeline = util.promisify(stream.pipeline);
 
-const defaultHeaders = {
-  'User-Agent': '@temporalio/testing installer',
-};
-
-const { GITHUB_TOKEN } = process.env;
-
-if (GITHUB_TOKEN) {
-  console.log(`Using GITHUB_TOKEN`);
-  defaultHeaders['Authorization'] = `Bearer ${GITHUB_TOKEN}`;
-}
-
-const latestReleaseRes = await got('https://api.github.com/repos/temporalio/sdk-java/releases/latest', {
+const options = {
   headers: {
-    ...defaultHeaders,
-    Accept: 'application/vnd.github.v3+json',
+    'User-Agent': '@temporalio/testing installer',
   },
-}).json();
-
-function findTestServerAsset(assets) {
-  for (const asset of assets) {
-    const m = asset.name.match(/^temporal-test-server_[^_]+_([^_]+)_([^.]+)\.(?:zip|tar.gz)$/);
-    if (m) {
-      const [_, assetPlatform, _assetArch] = m;
-      if (assetPlatform === systemPlatform) {
-        // TODO: assetArch === systemArch (no arm builds for test server yet)
-        return asset;
-      }
-    }
-  }
-  throw new Error(`No prebuilt test server for ${systemPlatform}-${systemArch}`);
-}
+  searchParams: { 'sdk-name': 'typescript', 'sdk-version': pkg.version },
+};
 
-const asset = findTestServerAsset(latestReleaseRes.assets);
-console.log('Downloading test server', { asset: asset.name, outputPath });
+// TODO: use "real" arch when we get arm builds for test server
+const assetUrl = `https://temporal.download/temporal-test-server/temporal-test-server_default_${systemPlatform}_amd64.${assetExt}`;
```

Comment:
How often does this run? Only on upgrade? Can this overwrite files? I assume we don't care about downloading something we already have?